EnglishTranslation-master/ETC.tsv

```diff
@@ -16320,7 +16320,7 @@ ETC_20151022_016321	Lighting up the bonfire
 ETC_20151022_016322	You've already used it
 ETC_20151022_016323	Finally, Orsha!
 ETC_20151022_016324	You've helped the settlers by lighting the fire!
-ETC_20151022_016325	You've obtained the usable objects!
+ETC_20151022_016325	Collected Useful Objects!
```

Comment:
![screenshot_20160302_00028](https://cloud.githubusercontent.com/assets/13687535/13513240/b853d38c-e143-11e5-860f-4a03030f7d03.jpg)

It says "usable objects" but the item name is "Useful Objects".   Should it be "You've obtained the Useful Objects!"?